icons/awwwards.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Awwwards</title><path d="m14.72 5.6-2.24 8.68-2.12-8.68H7.47l-2.12 8.68L3.11 5.6H0l4.01 12.65h2.74l2.17-8.18 2.16 8.18h2.74L17.83 5.6Zm7.19 12.8c1.2 0 2.09-.9 2.09-2.1s-.9-2.12-2.1-2.12-2.08.9-2.08 2.11.9 2.11 2.09 2.11"/></svg>
```

Comment:
Thank you @stefanobartoletti for your contribution! I can see some very little differences comparing your version in red with the orignal in black. It might be a precision issue only. Can you try with an increased precision?
Ex.
![image](https://github.com/simple-icons/simple-icons/assets/12817388/45fc20e8-51e2-49a5-b914-b863439da63c)
![image](https://github.com/simple-icons/simple-icons/assets/12817388/26b87da2-3651-42d9-b53f-c2bef8450f00)

Reply:
I will try to check this as soon as possible, I'm quite busy with regular work lately.

Reply:
Are you still working on this @stefanobartoletti?

Reply:
To be honest, I forgot about it :D

I've been quite busy lately, but I will try to check it

Reply:
I have updated the icon. The only edit from the one take from the original website is a resize to make it fit in the 24x24 canvas